src/noc/containers/noc-case-ref/noc-case-ref.component.scss

```diff
@@ -0,0 +1,3 @@
+:host ::ng-deep .govuk-input  {
+    max-width: 23ex;
```

Comment:
https://www.w3schools.com/cssref/css_units.asp
Not sure its a valid unit

Reply:
its valid unit but need to ensure adnan meant this